server/views/layouts/application.pug

```diff
@@ -8,10 +8,10 @@ html(lang="en").h-100
   body.h-100.d-flex.flex-column
     nav.navbar.navbar-expand-lg.navbar-light.bg-light.justify-content-between
       a.navbar-brand(href='/') Taskmine
-      if reply.request.currentUser
-        ul.navbar-nav.mr-auto
-          li.nav-item
-            a.nav-link(href='/users') Users
+      ul.navbar-nav.mr-auto
+        li.nav-item
+          a.nav-link(href='/users') Users
+      if reply.request.user
```

Comment:
use the force luke
https://github.com/hexlet-boilerplates/fastify-nodejs-application/blob/5243f56112b797e159ac2109d6ed588d060c777d/server/index.js#L79-L85

https://github.com/hexlet-boilerplates/fastify-nodejs-application/blob/5243f56112b797e159ac2109d6ed588d060c777d/server/views/layouts/application.pug#L20

Reply:
thanks